### PR TITLE
add config for testing the PR using wasser

### PR DESCRIPTION
**Pull request by kshtsk**
Signed-off-by: Kyr Shatskyy <kyrylo.shatskyy@suse.com>

### CODE REVIEW

**Comment by kshtsk** [comment]
jenkins run setup-vagrant

**Comment by kshtsk** [comment]
jenkins run setup-vagrant

**Comment by kshtsk** [comment]
jenkins run setup-vagrant

**Comment by kshtsk** [comment]
jenkins run setup-vagrant

**Comment by kshtsk** [comment]
jenkins run setup-vagrant

**Comment by kshtsk** [comment]
jenkins run setup-vagrant

**Comment by kshtsk** [comment]
jenkins run setup-vagrant

File: images/microOS/config.xml [hunk 1]
```diff
@@ -580,6 +580,7 @@
         <package name="python3-aiofiles"/>
         <package name="python3-requests"/>
         <package name="python3-rados"/>
+        <package name="python3-websockets"/>
```
Comment (jhesketh) [inline images/microOS/config.xml:583]: Why is this required? Aren't we installing the dependencies from the requirements.txt? (I'm making this assumption based on things like fastapi missing).

**Review by jhesketh — CHANGES_REQUESTED** [review]
lgtm, just one question relating to the requirements.

File: images/microOS/config.xml (hunk 1 above)
Comment (kshtsk) [reply images/microOS/config.xml:583]: @jhesketh that's question is not to me, and since it is not related to this particular patch in order to not block this PR I suggest you discuss this with @jecluis, @tserong, and the rest of the team on our slack channel.

Comment (jhesketh) [reply images/microOS/config.xml:583]: @kshtsk why is it in this pull request though?

Comment (jecluis) [reply images/microOS/config.xml:583]: yeah, lets drop it from this PR and have it merged with #269

**Comment by kshtsk** [comment]
jenkins run setupvagrant

**Comment by kshtsk** [comment]
jenkins run setupvagrant

**Comment by kshtsk** [comment]
jenkins run setupvagrant

**Comment by kshtsk** [comment]
@jecluis I need an approve to merge this.

**Comment by kshtsk** [comment]
This does not cover Tumbleweed, surprise Tumbleweed OpenStack JeOS image does not have configured wheel group in comparison to Leap.

**Comment by jecluis** [comment]
> This does not cover Tumbleweed, surprise Tumbleweed OpenStack JeOS image does not have configured wheel group in comparison to Leap.

What does this mean in practice? Is this a blocker to make wasser work?

**Comment by kshtsk** [comment]
> > This does not cover Tumbleweed, surprise Tumbleweed OpenStack JeOS image does not have configured wheel group in comparison to Leap.
> 
> What does this mean in practice? Is this a blocker to make wasser work?

yes, since we decided repo agnostic workflow config

File: images/microOS/config.xml (hunk 1 above)
Comment (tserong) [reply images/microOS/config.xml]: > Why is this required? Aren't we installing the dependencies from the requirements.txt? (I'm making this assumption based on things like fastapi missing).

Actually some of this stuff is a little bit messy ATM -- images/microOS/config.sh is doing `pip install fastapi uvicorn` rather than using requirements.txt. I'm trying to clean this up as part of #91

Comment (kshtsk) [reply images/microOS/config.xml]: I don't understand why do we continue discussing it in this PR where the change already dropped.

**Review by jecluis — APPROVED** [review]

File: images/microOS/config.xml (hunk 1 above)
Comment (tserong) [reply images/microOS/config.xml]: Josh asked a question and I thought it might be helpful to answer it.
